webextensions/native-messaging-host/build_pkg.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+[ ! -f ./build_pkg_configs.sh ] || exit 1
+source ./build_pkg_configs.sh

```

Comment:
```suggestion
. ./build_pkg_configs.sh
```
`source`はbashismなので、`.`にするかshebangを`!/bin/bash`にするかした方が良さそうです。

Reply:
こちら訂正しました